libs/ui/src/theme/ThemeColorVars.tsx

```diff
@@ -5,6 +5,153 @@ import { css } from 'styled-components/macro'
 
 import { UI } from '../enum'
 
+interface ChainAccentConfig {
+  bgVar: UI
+  borderVar: UI
+  accentVar?: UI
+  lightBg: string
+  darkBg: string
+  lightBorder: string
+  darkBorder: string
+  lightColor: string
+  darkColor: string
+}
+
+interface ChainAccentInput {
+  bgVar: UI
+  borderVar: UI
+  accentVar?: UI
+  color: string
+  lightColor?: string
+  darkColor?: string
+  lightBgAlpha?: number
+  darkBgAlpha?: number
+  lightBorderAlpha?: number
+  darkBorderAlpha?: number
+}
+
+const CHAIN_LIGHT_BG_ALPHA = 0.22
+const CHAIN_DARK_BG_ALPHA = 0.32
+const CHAIN_LIGHT_BORDER_ALPHA = 0.45
+const CHAIN_DARK_BORDER_ALPHA = 0.65
+
+const chainAlpha = (color: string, alpha: number): string => transparentize(color, 1 - alpha)
+
+function createChainAccent({
+  bgVar,
+  borderVar,
+  accentVar,
+  color,
+  lightColor = color,
+  darkColor = color,
+  lightBgAlpha = CHAIN_LIGHT_BG_ALPHA,
+  darkBgAlpha = CHAIN_DARK_BG_ALPHA,
+  lightBorderAlpha = CHAIN_LIGHT_BORDER_ALPHA,
+  darkBorderAlpha = CHAIN_DARK_BORDER_ALPHA,
+}: ChainAccentInput): ChainAccentConfig {
+  return {
+    bgVar,
+    borderVar,
+    accentVar,
+    lightBg: chainAlpha(lightColor, lightBgAlpha),
+    darkBg: chainAlpha(darkColor, darkBgAlpha),
+    lightBorder: chainAlpha(lightColor, lightBorderAlpha),
+    darkBorder: chainAlpha(darkColor, darkBorderAlpha),
+    lightColor,
+    darkColor,
+  }
+}
+
+const CHAIN_ACCENT_CONFIG: ChainAccentConfig[] = [
```

Comment:
`CHAIN_ACCENT_CONFIG` should be `Record<SupportedChainId, ChainAccentConfig>`. That way Typescript will protect us from mistakes